scripts/usb_reset.py

```diff
@@ -90,6 +89,48 @@ def get_cg_dir(domid):
     return "/sys/fs/cgroup/devices/qemu-{}".format(domid)
 
 
+def get_ids_path(device):
+    usb_dir = "/var/run/nonpersistent/usb"
+    if not os.path.isdir(usb_dir):
+        os.makedirs(usb_dir)
+    return usb_dir + "/{}".format(device)
```

Comment:
You can use os.path.join for this:

    return os.path.join(usb_dir, device)

Reply:
done.